service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/model/Model.scala

```diff
@@ -118,6 +119,16 @@ case class OsnCertificates(
     serverTlsCert: String
 )
 
+case class ApplicationDescriptor(
+    applicationName: String,
+    applicationType: String = "",
+    properties: Array[Property],
+    roles: Array[String] = Array.empty[String],
+    initArgsNames: Array[String] = Array.empty[String],
```

Comment:
lets remove this from here for now (i.e. deprecate chain-codes with non-empty init args for bundling in application)

Reply:
done.